index.html

```diff
@@ -149,7 +149,7 @@ <h1 id="title">RDF-star Working Group Charter</h1>
       </div>
 
       <section id="background" class="background issue">
-        <p>RDF-star (initially spelled RDF*) is an extension of RDF that was initially proposed by Olaf Hartig and Bryan Thompson in 2014, providing a concise way to make statements about statements.It has become quite popular, especially as a mean to provide interoperability between RDF and Property Graphs, and has been already implemented to various extent by <a href="https://w3c.github.io/rdf-star/implementations.html">a number of vendors and open-source libraries</a>. A <a href="https://w3c.github.io/rdf-star/">subgroup</a> of the <a href="https://www.w3.org/community/rdf-dev/">RDF-DEV Community Group</a> then formed to gather feedback, <a href="https://w3c.github.io/rdf-star/UCR/rdf-star-ucr.html">use-cases</a>, and explore the possibility of <a href="https://www.w3.org/2021/12/rdf-star.html">integrating RDF-star</a> into the existing RDF standards.</p>
+        <p>RDF-star (formerly spelled RDF*) is an extension of RDF that was initially proposed by Olaf Hartig and Bryan Thompson in 2014, providing a concise way to make statements about statements.It has become quite popular, especially as a mean to provide interoperability between RDF and Property Graphs, and has been already implemented to various extent by <a href="https://w3c.github.io/rdf-star/implementations.html">a number of vendors and open-source libraries</a>. A <a href="https://w3c.github.io/rdf-star/">subgroup</a> of the <a href="https://www.w3.org/community/rdf-dev/">RDF-DEV Community Group</a> then formed to gather feedback, <a href="https://w3c.github.io/rdf-star/UCR/rdf-star-ucr.html">use-cases</a>, and explore the possibility of <a href="https://www.w3.org/2021/12/rdf-star.html">integrating RDF-star</a> into the existing RDF standards.</p>
```

Comment:
Spotted a couple other things in this line of code which may as well go in the same PR...
```suggestion
        <p>RDF-star (formerly spelled RDF*) is an extension of RDF that was initially proposed by Olaf Hartig and Bryan Thompson in 2014, providing a concise way to make statements about statements. It has become quite popular, especially as a mean to provide interoperability between RDF and Property Graphs, and has already been implemented to various extents by <a href="https://w3c.github.io/rdf-star/implementations.html">a number of vendors and open-source libraries</a>. A <a href="https://w3c.github.io/rdf-star/">subgroup</a> of the <a href="https://www.w3.org/community/rdf-dev/">RDF-DEV Community Group</a> then formed to gather feedback, <a href="https://w3c.github.io/rdf-star/UCR/rdf-star-ucr.html">use-cases</a>, and explore the possibility of <a href="https://www.w3.org/2021/12/rdf-star.html">integrating RDF-star</a> into the existing RDF standards.</p>
```